templates/worker/00-worker/vsphere/files/vsphere-non-virtual-ip.yaml

```diff
@@ -1,142 +0,0 @@
-filesystem: "root"
-mode: 0755
-path: "/usr/local/bin/non_virtual_ip"
-contents:
-  inline: |
-    {{ if .Infra.Status.PlatformStatus.VSphere -}}
```

Comment:
@celebdor this _probably_ won't be a problem but for UPI in vSphere we don't want any "baremetal networking" services coming on line.

Reply:
I removed the other platforms from the change. Still, if you decide to adopt this change for IPv6, I think making the `units` optional but leave the scripts, should be reasonable.